src/main/java/org/codehaus/mojo/flatten/FlattenMojo.java

```diff
@@ -252,6 +252,14 @@
     @Parameter( required = false )
     private FlattenDescriptor pomElements;
 
+    /**
+     * Dictates whether dependency exclusions stanzas should be included in the flattened POM. By default exclusions
+     * will be included in the flattened POM but if you wish to omit exclusions stanzas from being present then set
+     * this configuration property to <code>true</code>.
+     */
+    @Parameter( defaultValue = "false", required = false )
```

Comment:
Please add `@since 1.3.0`

Reply:
Thank you for the review @slawekjaranowski  - I have added the @since declaration.